MAUI/Autocomplete/Header-and-Footer.md

```diff
@@ -0,0 +1,273 @@
+                            HorizontalOptions="Center" 
+                            VerticalOptions="Center" 
+                            TextColor="#006bcd"   />
+                </StackLayout>
+            </autocomplete:SfAutoComplete.DropDownHeaderView>
+        </autocomplete:SfAutoComplete>
+    </StackLayout>
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+using Syncfusion.Maui.Inputs;
+using System.Collections.Generic;
+
+namespace AutocompleteSample
+{
+    public partial class MainPage : ContentPage
+    {
+        Label SearchLabel;
+        public MainPage()
+        {
+            InitializeComponent();
+            StackLayout stackLayout = new StackLayout()
+            {
+                VerticalOptions = LayoutOptions.Start,
+                HorizontalOptions = LayoutOptions.Start,
+                Padding = new Thickness(30)
+            };
+
+            SfAutoComplete autoComplete = new SfAutoComplete()
+            {
+                HeightRequest = 40,
+                ShowDropDownHeaderView = true,
+                DropDownHeaderViewHeight = 50,
+                AutoCompleteSource = new List<string>()
+                {
+                    "India",
+                    "Uganda",
+                    "Ukraine",
+                    "Canada",
+                    "United Arab Emirates",
+                    "France",
+                    "United Kingdom",
+                    "China",
+                    "United States",
+                    "Japan",
+                    "Angola"
+                }
+            };
+
+            StackLayout layout = new StackLayout()
+            {
+                BackgroundColor = Color.FromHex("#f0f0f0")
+            };
+
+            SearchLabel = new Label()
+            {
+                FontSize = 20,
+                VerticalTextAlignment = TextAlignment.Center,
+                HorizontalOptions = LayoutOptions.Center,
+                VerticalOptions = LayoutOptions.Center,
+                TextColor = Color.FromHex("#006bcd")
+            };
+
+            layout.Children.Add(SearchLabel);
+            autoComplete.DropDownHeaderView = layout;
+            autoComplete.ValueChanged += SfAutoComplete_ValueChanged;
+            stackLayout.Children.Add(autoComplete);
+            this.Content = stackLayout;
+        }
+    }
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Events 
+
+The following event will be hooked.
+
+{% tabs %}
+
+{% highlight C# %}
+
+        private void SfAutoComplete_ValueChanged(object sender, ValueChangedEventArgs e)
+        {
+            SearchLabel.Text = "Search for " + e.Value;
+        }
+        
+{% endhighlight %}
+
+{% endtabs %}
+
+## Footer Content
+
+We can provide Footer Content at the bottom of the AutoComplete's Suggestion box. [`DropDownFooterView`] property is used to set the content of the footer.The height of the Header in the [`SfAutoComplete`] can be adjusted by the property [`DropDownFooterViewHeight`].
```

Comment:
Need to enable the ShowDropDownFooterView property to show the footer view, please notify it.